Updates/841_Saltgurka-Honor_Hold_Updates.sql

```diff
@@ -0,0 +1,1092 @@
+-- ----------------------------------------------------------
+-- Model and equipment updates
+-- ----------------------------------------------------------
+-- Update EquipmentTemplate for all Honor Hold Defenders. Entry 20513 (the two guards next to the destroyed keep) had the correct ones, all others were wrong for some reason.
+UPDATE `creature_template` SET `EquipmentTemplateId`=1195 WHERE `entry`=16842;
+
+-- Also the human model was missing from their creature_template
+UPDATE `creature_template` SET `ModelId1`=16387,`ModelId2`=16388,`ModelId3`=16389,`ModelId4`=16390 WHERE `entry` IN (16842,20513); -- 16390
+
+-- Removing the gender link, because some creatures need to use the male model only
+UPDATE `creature_model_info` SET `modelid_other_gender`=0 WHERE `modelid` BETWEEN 16387 AND 16390;
+
+-- The guards outside the destroyed keep should always be male, as indicated by their texts. "I am a grumpy old man!"
+UPDATE `creature` SET `modelid`=16387 WHERE `guid` IN(72636,72637);
+
+-- Nethergarde Infantry were missing the female model
+UPDATE `creature_template` SET `ModelId1`=16377,`ModelId2`=16376,`ModelId3`=16375 WHERE `entry`=16831; 
+UPDATE `creature_model_info` SET `modelid_other_gender`=0 WHERE `modelid` IN(16375,16376,16377);
+UPDATE `creature_model_info` SET `modelid_alternative`=16377 WHERE `modelid`=16376; 
+
+-- Nethergarde infantry inside inn should have mugs and mutton in their hands.
+DELETE FROM `creature_equip_template` WHERE `entry` = 5593;
+INSERT INTO `creature_equip_template` (`entry`, `equipentry1`, `equipentry2`, `equipentry3`) VALUES
+(5593,2704,13407,0);

```

Comment:
Currently the rule of thumb is to try allocate equip template from lowest index, because @X-Savior started making a decollisioning voa-side, which sometime in the future i will try to finish (or will tell someone else to finish it) so that we can unify cross-db IDs. So please always allocate the lowest free index :) I will then check it with voa db just for merge errors. The reason why this is complicated is that I am the only DB-provider who has both DBs and can export Voa SQL. Sorry for being so uptight about this, but it saves me time.